modules/cnv-cloning-pvc-of-vm-disk-into-new-datavolume.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume.adoc
+
+[id="cnv-cloning-pvc-of-vm-disk-into-new-datavolume_{context}"]
+= Cloning the PersistentVolumeClaim of a virtual machine disk into a new DataVolume
+
+You can clone a PVC of an existing virtual machine disk into a new DataVolume.
+The new DataVolume can then be used for a new virtual machine.
+
+[NOTE]
+====
+When a DataVolume is created independently of a virtual machine, the lifecycle
+of the DataVolume is independent of the virtual machine. If the virtual machine
+is deleted, neither the DataVolume nor its associated PVC will be deleted.
+====
+
+.Prerequisites
+* A PVC of an existing virtual machine disk. The associated virtual machine must
```

Comment:
This list isn't parallel. Maybe "Determine the PVC of an existing virtual machine disk to use. You must power down the virtual machine that is associated with the PVC before you can clone it."